users/serializers.py

```diff
@@ -15,11 +15,33 @@
 
 
 class DataOceanUserSerializer(serializers.ModelSerializer):
+
+    def validate(self, data):
+        """
+        validate fop and legal_entity fields
+        """
```

Comment:
short docstring can be located in one line
```suggestion
        """ validate fop and legal_entity fields """
```